beta/src/pages/learn/start-a-new-react-project.md

```diff
@@ -1,41 +1,41 @@
 
 </Intro>
 
-## Choose your own adventure {/*choose-your-own-adventure*/}
+## अपना खुद का रोमांच चुनें {/*अपना खुद का रोमांच चुनें*/}
 
-React is a library that lets you organize UI code by breaking it apart into pieces called components. React doesn't take care of routing or data management. For these features, you'll want to use third-party libraries or write your own solutions. This means there are several ways to start a new React project:
+React एक library है जो आपको यूआई कोड को कौम्पोनॅन्ट नामक टुकड़ों में तोड़कर व्यवस्थित करने देता है। React रूटिंग या डेटा प्रबंधन का ध्यान नहीं रखता है। इन सुविधाओं के लिए, आप थर्ड-पार्टी लाइब्रेरी का उपयोग करना चाहेंगे या अपने स्वयं के समाधान लिखना चाहेंगे। इसका मतलब है कि एक नया React प्रोजेक्ट शुरू करने के कई तरीके हैं:
 
-* Start with a **minimal set up with just a toolchain,** adding features to your project as necessary.
-* Start with an **opinionated framework** with common functionality already built in.
+* केवल एक टूलचेन के साथ **न्यूनतम सेट अप के साथ प्रारंभ करें,** आवश्यकतानुसार अपने प्रोजेक्ट में सुविधाएँ जोड़ें।
+* पहले से निर्मित सामान्य कार्यक्षमता के साथ **विचारित ढांचे** के साथ प्रारंभ करें।
 
-Whether you're just getting started, looking to build something big, or want to set up your own toolchain, this guide will set you on the right path.
+चाहे आप अभी शुरुआत कर रहे हैं, कुछ बड़ा बनाना चाहते हैं, या अपनी खुद की टूलचेन स्थापित करना चाहते हैं, यह मार्गदर्शिका आपको सही रास्ते पर ले जाएगी।
 
-## Getting started with a React toolchain {/*getting-started-with-a-react-toolchain*/}
+## React टूलचेन के साथ शुरुआत करना {/*आरंभ करना-के साथ-साथ-प्रतिक्रिया-टूलचेन*/}
 
-If you're just getting started with React, we recommend [Create React App](https://create-react-app.dev/), the most popular way to try out React's features and a great way to build a new single-page, client-side application. Create React App is an unopinionated toolchain configured just for React. Toolchains help with things like:
+यदि आप React के साथ अभी शुरुआत कर रहे हैं, तो हम अनुशंसा करते हैं [Create React App](https://create-react-app.dev/), React की विशेषताओं को आज़माने का सबसे लोकप्रिय तरीका और एक नया सिंगल-पेज, क्लाइंट-साइड एप्लिकेशन बनाने का एक शानदार तरीका । Create React App सिर्फ React के लिए कॉन्फ़िगर किया गया एक अप्रकाशित टूलचैन है। टूलचेन इस तरह की चीजों में मदद करते हैं:
 
-* Scaling to many files and components
-* Using third-party libraries from npm
-* Detecting common mistakes early
-* Live-editing CSS and JS in development
-* Optimizing the output for production
+* कई फाइलों और कौम्पोनॅन्ट के लिए स्केलिंग
+* npm . से थर्ड-पार्टी लाइब्रेरी का उपयोग करना
+* सामान्य गलतियों का जल्द पता लगाना
+* विकास में CSS और JS का लाइव संपादन
```

Comment:
`विकास में` => `डेवलपमेंट के समय`

let me know if you have any other suggestions for this

Reply:
डेवलपमेंट के समय looks good here